frontend-ssr/components/cli_git/match_commands.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+class MatchWords extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: '',
+            message: '',
+        }
+
+        this.handleInputChange = this.handleInputChange.bind(this);
```

Comment:
Bind is ok but with a lot of methods things get a little messy.

I suggest you declare the custom methods on the class like this:

```
handleSubmit  = (event) => {
  // ...
}
```

This way **this** will not get lost.